xapian-core/examples/simpleexpand.cc

```diff
@@ -26,7 +26,7 @@
 
 #include <iostream>
 #include <string>
-
+#include <parseint.h>
```

Comment:
An in-tree header should be included as `"parseint.h"` not `<parseint.h>` (`""` vs `<>` affects the header search path).

But I wonder if we're actually better off just leaving the "simple" examples alone.  Currently they can be built stand-alone using an installed xapian-core, but this change means that other Xapian source files would be required.